natgeo/mortar
# [SYTO-1223] Added some tweaks from Kevin's feedback

To test:

- Verify tweaks from [Kevin's feedback](https://jira.natgeo.com/browse/SYTO-1223?focusedCommentId=440209&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-440209) are met.
